setup.py

```diff
@@ -37,10 +37,12 @@
 MAINTAINER_EMAIL = 'holmgren@email.arizona.edu'
 URL = 'https://github.com/pvlib/pvlib-python'
 
-INSTALL_REQUIRES = ['numpy >= 1.10.4',
+INSTALL_REQUIRES = ['numpy >= 1.13.3',
```

Comment:
what are you using that requires numpy 1.13? Increasing the required version is allowed, but I'd like to understand why.